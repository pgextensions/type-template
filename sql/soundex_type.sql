CREATE EXTENSION "soundex-type";

CREATE TABLE test_table (
  id INT,
  data soundex
);

INSERT INTO test_table VALUES (1, 'hello');
INSERT INTO test_table VALUES (2, 'world');

INSERT INTO test_table VALUES (3, 'john');
INSERT INTO test_table VALUES (4, 'joan');

SELECT * FROM test_table;

DROP TABLE test_table;
