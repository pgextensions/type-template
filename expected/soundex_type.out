CREATE EXTENSION "soundex-type";
CREATE TABLE test_table (
  id INT,
  data soundex
);
INSERT INTO test_table VALUES (1, 'hello');
INSERT INTO test_table VALUES (2, 'world');
INSERT INTO test_table VALUES (3, 'john');
INSERT INTO test_table VALUES (4, 'joan');
SELECT * FROM test_table;
 id | data 
----+------
  1 | H400
  2 | W643
  3 | J500
  4 | J500
(4 rows)

DROP TABLE test_table;
